zipline/data/us_equity_pricing.py

```diff
@@ -210,8 +210,11 @@ class BcolzDailyBarWriter(object):
     def __init__(self, filename, calendar, start_session, end_session):
         self._filename = filename
 
-        assert calendar.is_session(start_session), "Start session is invalid!"
-        assert calendar.is_session(end_session), "End session is invalid!"
+        if start_session != end_session:
+            assert calendar.is_session(start_session),\
```

Comment:
Can we make these a `ValueError` and show the session in the message. In general, assertions should be used to validate invariants in code that do not depend on user inputs. An assertion error should indicate that there is a bug in the program, but not that a function was called with invalid arguments.